src/processes.c

```diff
@@ -2097,6 +2102,8 @@ static int ps_read (void)
 	ps_submit_state ("stopped",  stopped);
 	ps_submit_state ("paging",   paging);
 	ps_submit_state ("blocked",  blocked);
+	ps_submit_state ("no_cmdline",  (count - num_cmdline));
```

Comment:
I believe these were also kept aligned. Same comment (1 space after `,` here and adjust all of the others).

Reply:
Done.